tests/unit/mainwindow/test_tabwidget.py

```diff
@@ -128,6 +129,19 @@ def test_update_tab_titles_benchmark(self, benchmark, widget,
 
         benchmark(widget.update_tab_titles)
 
+    def test_tab_min_width(self, widget, fake_web_tab, config_stub, qtbot):
+        widget.addTab(fake_web_tab(), 'foobar')
+        widget.addTab(fake_web_tab(), 'foobar1')
+        min_size = widget.tabBar().tabRect(0).width() + 10
+        config_stub.val.tabs.min_width = min_size
+        assert widget.tabBar().tabRect(0).width() == min_size
```

Comment:
In the process of testing this, I found out that `tabRect` will refuse to give values larger than the tabBar if only `minimumTabSizeHint` is larger than the tabbar, but it will give larger values if both `tabSizeHint` and `minimumTabSizeHint` are larger. Just some odd behavior I found...